torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3319,24 +3319,44 @@ def sample_inputs_conv2d(op_info, device, dtype, requires_grad, jit_fail_sample=
 def sample_inputs_group_norm(opinfo, device, dtype, requires_grad, **kwargs):
     make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
 
-    # Ordered as input shape, num groups, and eps
+    # Ordered as input shape, num groups, and kwargs for eps
     cases: Tuple[Tuple[int], int, float] = (  # type: ignore[assignment]
-        ((1, 6, 3), 2, 0.5),
-        ((2, 6, 3), 2, -0.5),
-        ((1, 2), 1, None),
-        ((0, 2), 1, None),
+        ((20, 6, 10, 10), 3, {'eps' : 1e-5}),
```

Comment:
These new cases are cool but put them in a new `reference_inputs_group_norm`. See an example here:

https://github.com/pytorch/pytorch/blob/dd43903fa99b8549225ec63c2e81ef4693436be0/torch/testing/_internal/common_methods_invocations.py#L1001

And then the association with the OpInfo is like this:

https://github.com/pytorch/pytorch/blob/dd43903fa99b8549225ec63c2e81ef4693436be0/torch/testing/_internal/common_methods_invocations.py#L8373

The "reference inputs" generator is special and only used by tests who want a larger set of sample inputs.